### PR TITLE
Release/0.6.0

### PR DESCRIPTION
**Pull request by asoltysik**
0.6.0-M2 up on maven central

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,33 @@
+Version 0.6.0 (2018-08-29)
```
Comment (BenFradet) [inline CHANGELOG]: date

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+Drop support for Scala 2.10 (closes #75)
+Make Scala 2.12 the default version (closes #80)
+Add issue template (closes #83)
+Add pull request template (closes #84)
+Add CONTRIBUTING.md (closes #85)
+Integrate scalafmt (closes #86)
+Remove vagrant setup (closes #73)
+Remove cruft from .gitignore (closes #81)
+
+
```
Comment (BenFradet) [inline CHANGELOG]: extraneous empty line

**Comment by asoltysik** [comment]
Travis still failing sometimes because of https://github.com/travis-ci/travis-ci/issues/10053

File: modules/id-emitter/src/main/scala/com/snowplowanalytics/snowplow/scalatracker/emitters/id/BaseEmitter.scala [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package com.snowplowanalytics.snowplow.scalatracker.emitters.id
```
Comment (BenFradet) [inline modules/id-emitter/src/main/scala/com/snowplowanalytics/snowplow/scalatracker/emitters/id/BaseEmitter.scala:13]: header

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,33 @@
+Version 0.6.0 (2018-08-29)
+--------------------------
+Track page URL in the tracker subject (closes #92)
```
Comment (chuwy) [inline CHANGELOG]: Please remove `closes`. Should be just `(#XX)`.

**Review by chuwy — COMMENTED** [review]
LGTM

File: modules/id-emitter/src/main/scala/com/snowplowanalytics/snowplow/scalatracker/emitters/id/BaseEmitter.scala (hunk 3 above)
Comment (asoltysik) [reply modules/id-emitter/src/main/scala/com/snowplowanalytics/snowplow/scalatracker/emitters/id/BaseEmitter.scala:13]: done

**Review by BenFradet — APPROVED** [review]
I managed to publish referer parser yesterday, you just need to retry until it works :D

**Pull request merged**